### PR TITLE
Fix colliding with builtin function

### PR DESCRIPTION
**Pull request by huynq0911**
Some variable name, function name is colliding with builtin function.
File changed:
- src/common/utils/clair/utils_test.go
- src/common/utils/ldap/ldap.go
- src/core/promgr/pmsdriver/admiral/admiral_test.go
- src/core/service/notifications/clair/handler.go
- src/jobservice/runtime/bootstrap.go
- src/replication/trigger/cache.go

### CODE REVIEW

File: src/core/service/notifications/clair/handler.go [hunk 1]
```diff
@@ -64,8 +64,8 @@ func (h *Handler) Handle() {
 			ns[vuln.NamespaceName] = true
 		}
 	}
-	if new := notification.New; new != nil {
-		if vuln := new.Vulnerability; vuln != nil {
+	if newNotification := notification.New; newNotification != nil {
```
Comment (reasonerjt) [inline src/core/service/notifications/clair/handler.go]: I'm not a big fan of long variable name.  Could you let me know why this is necessary?

**Comment by reasonerjt** [comment]
@huynq0911 Thanks for pointing it out, did you use any tool to find out the collisions?

**Comment by huynq0911** [comment]
> @huynq0911 Thanks for pointing it out, did you use any tool to find out the collisions?

Yes, I used GoLand IDE.  It shows the warning.

**Comment by wy65701436** [comment]
thanks, @huynq0911, is it possible to add a rule to check this in make and avoid any new colliding?

**Comment by huynq0911** [comment]
> thanks, @huynq0911, is it possible to add a rule to check this in make and avoid any new colliding?

@wy65701436: hmm, the default rule in Goland IDE already supports to check this warning. Once you create new code with the collision, the IDE will warn it.

**Comment by wy65701436** [comment]
> > thanks, @huynq0911, is it possible to add a rule to check this in make and avoid any new colliding?
> 
> @wy65701436: hmm, the default rule in Goland IDE already supports to check this warning. Once you create new code with the collision, the IDE will warn it.
> Besides, developers can check it from this to avoid colliding when set variable/function name: https://golang.org/pkg/builtin/

To let IDE give warnings cannot block new collisions, I mean we can figure out a way to check this in the Makefile, and then can fail PR by caching collisions.

Anyway, let's merge this PR first.

**Comment by huynq0911** [comment]
> > > thanks, @huynq0911, is it possible to add a rule to check this in make and avoid any new colliding?
> > 
> > 
> > @wy65701436: hmm, the default rule in Goland IDE already supports to check this warning. Once you create new code with the collision, the IDE will warn it.
> > Besides, developers can check it from this to avoid colliding when set variable/function name: https://golang.org/pkg/builtin/
> 
> To let IDE give warnings cannot block new collisions, I mean we can figure out a way to check this in the Makefile, and then can fail PR by caching collisions.
> 
> Anyway, let's merge this PR first.

That's an option. Thanks!

**Pull request merged**